app/models/subcase.js

```diff
@@ -27,7 +27,7 @@ export default ModelWithModifier.extend({
     inverse: null,
   }),
   submissionActivities: hasMany('submission-activity', {
-    serialize: true,
+    serialize: false,
```

Comment:
Dat zou ik dan precies eens willen testen.
We schrijven op andere plaatsen ook die serialize: false, maar die situaties waar dat wel nodig is zijn niet duidelijk.
in de `/serializers/application.js` staat niet expliciet vermeld wat de defaults zijn.
Volgens de ember-documenatie:
https://api.emberjs.com/ember-data/4.1/classes/JSONAPISerializer/methods/shouldSerializeHasMany?anchor=shouldSerializeHasMany
`By default only many-to-many and many-to-none relationships are serialized. This could be configured per relationship by Serializer's attrs object.`

Dat dit laten staan verwarring zou geven ben ik niet helemaal mee eens. De bedoeling van de optie is wel duidelijk, doch mogelijk overbodig.

Ik wil het gerust nog wel eens bekijken maar zou geen blocker mogen zijn voor deze PR als de boolean op false zetten of verwijderen hetzelfde doet.
Dit kunnen we achteraf dan nog aanpassen.